CharacterMap/CharacterMap/Views/CalligraphyView.xaml

```diff
@@ -451,7 +451,7 @@
                                 <InkToolbarBallpointPenButton
                                     x:Name="BallpointPen"
                                     core:Properties.ClickAnimationOffset="0.87"
-                                    core:Properties.PointerOverAnimation="Content"
+                                    core:Properties.PointerOverAnimation="Content,ContentFillGlyph"
                                     core:Properties.PointerPressedAnimation="Content|Scale"
```

Comment:
Adding the `ContentFillGlyph` breaks the animation

Is this wrong?
```xml
<InkToolbarBallpointPenButton
    ...
    core:Properties.PointerPressedAnimation="Content|ContentFillGlyph|Scale"
    ...
/>
```
Also @JohnnyWestlake, do you know why these two properties animations don't play on the checked states?

Sorry for the ping.

Reply:
CheckedState has a different visual, added it in

Reply:
Thanks, I was confused because they were fine on the `InkToolbarCustomPenButton`